### ISSUE TITLE
CDTS font errors in console, rejected by FireFox

### ISSUE THREAD

**Issue by TeamDman** [wet-boew/cdts-sgdc#116]
![image](https://user-images.githubusercontent.com/9356891/79252643-f8ce3080-7e4f-11ea-9c03-4b4e0334768a.png)
```
downloadable font: Failed to convert WOFF 2.0 font to SFNT (font-family: "Glyphicons Halflings" style:normal weight:400 stretch:100 src index:1) source: https://ssl-templates.services.gc.ca/app/cls/WET/gcintranet/v4_0_32/fonts/glyphicons-halflings-regular.woff2
downloadable font: rejected by sanitizer (font-family: "Glyphicons Halflings" style:normal weight:400 stretch:100 src index:1) source: https://ssl-templates.services.gc.ca/app/cls/WET/gcintranet/v4_0_32/fonts/glyphicons-halflings-regular.woff2
```
Glyphicons are still working in my app at least, but the console warnings are annoying...

Potentially relevant: 
Proxy misconfiguration causing fonts to raise errors
https://github.com/ember-cli/ember-cli/issues/1532

**Comment by patricehernert** [comment]
@TeamDman Could you confirm this issue still exists in version 4.0.39 ? Thanks.

**Comment by TeamDman** [comment]
![image](https://user-images.githubusercontent.com/9356891/103008694-898efd00-4503-11eb-9c2a-4156d00353f5.png)
Still seems to be present

**Comment by ahmad-shahid** [comment]
Font issue has been fixed in v4.0.43.

This can be confirmed via the latest sample pages. 
https://ssl-templates.services.gc.ca/app/cls/WET/gcintranet/v4_0_43/cdts/samples/index.html
